.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Dependencies
+
+on:
+  # Run every day.
+  schedule:
+    - cron: '0 3 * * *'
+  # And on on every PR merge so we get the updated dependencies ASAP, and to make sure the changelog doesn't conflict.gi
+  push:
+    branches:
+      - main
+
+jobs:
+  cli:
+    uses: getsentry/github-workflows/.github/workflows/updater.yml@v1
+    with:
+      path: sentry-cli.properties
+      name: CLI
+    secrets:
+      # If a custom token is used instead, a CI would be triggered on a created PR.
+      # api_token: ${{ secrets.CI_DEPLOY_KEY }}
+      api_token: ${{ github.token }}
```

Comment:
Maybe I missed it, but `sentry-cli` doesn't ship with `sentry-kmp`. 

The value from `sentry-cli.properties` is not used.

Reply:
yup, that's an oversight, I somehow assumed the action itself needed the properties file